namespaces/live-1.cloud-platform.service.justice.gov.uk/calculate-release-dates-api-dev/resources/variables.tf

```diff
@@ -0,0 +1,42 @@
+
+variable "cluster_name" {
+}
+
+variable "application" {
+  description = "Name of Application you are deploying"
+  default     = "Calculate release dates API"
+}
+
+variable "namespace" {
+  default = "calculate-release-dates-api-dev"
+}
+
+variable "business_unit" {
+  description = "Area of the MOJ responsible for the service."
+  default     = "HMPPS"
+}
+
+variable "team_name" {
+  description = "The name of your development team"
+  default     = "farsight-devs"
+}
+
+variable "environment" {
+  description = "The type of environment you're deploying to."
+  default     = "dev"
+}
+
+variable "infrastructure_support" {
+  description = "The team responsible for managing the infrastructure. Should be of the form team-email."
+  default     = "dps-hmpps@digital.justice.gov.uk"
+}
+
+variable "is_production" {
+  default = "false"
+}
+
+variable "slack_channel" {
+  description = "Team slack channel to use if we need to contact your team"
+  default     = "calculate_release_dates"
+}
```

Comment:
Do you have slack channels called `calculate_release_dates` and `calculate_release_dates_dev` , as mentioned in these files?  They can be created afterwards - it doesn't check here.

Reply:
I have created a dev channel for those, the other is our main public channel